src/components/Organization/Team.tsx

```diff
@@ -0,0 +1,160 @@
+    select: (data) => data.members,
+  })
+}
+
+// Fetch hook for pending members
+const usePendingTeamMembers = ({
+  options,
+}: {
+  options?: Omit<UseQueryOptions<PendingTeamMembersResponse>, 'queryKey' | 'queryFn'>
+} = {}) => {
+  const { bearedFetch, signerAddress } = useAuth()
+  return useQuery({
+    queryKey: ['organizations', 'members', 'pending', signerAddress],
+    queryFn: () =>
+      bearedFetch<PendingTeamMembersResponse>(
+        ApiEndpoints.OrganizationPendingMembers.replace('{address}', signerAddress)
+      ),
+    ...options,
+    select: (data) => data.pending,
+  })
+}
+
+// Reusable table component for both active and pending members
+const TeamMembersTable = ({ members, showExpiration = false }: { members: Member[]; showExpiration?: boolean }) => (
+  <TableContainer>
+    <Table>
+      <Thead>
+        <Tr>
+          <Th>
+            <Trans i18nKey='team.member_name'>Name</Trans>
+          </Th>
+          {showExpiration && (
+            <Th textAlign='center'>
+              <Trans i18nKey='team.expiration'>Expiration</Trans>
+            </Th>
+          )}
+          <Th textAlign='center'>
+            <Trans i18nKey='team.member_role'>Role</Trans>
+          </Th>
+        </Tr>
+      </Thead>
+      <Tbody>
+        {members.map((member, i) => (
+          <Tr key={i}>
+            <Td>
+              {member.info ? (
+                <Flex>
+                  <Avatar name={member.info.firstName} size='sm' />
+                  <Box ml='3'>
+                    <Text fontWeight='bold'>
+                      {member.info.firstName} {member.info.lastName}
+                    </Text>
+                    <Text fontSize='sm'>{member.info.email}</Text>
+                  </Box>
+                </Flex>
+              ) : (
+                <Text>{member.email}</Text>
+              )}
+            </Td>
+            {showExpiration && member.expiration && (
+              <Td textAlign='center'>
+                <Text>{new Date(member.expiration).toLocaleString()}</Text>
+              </Td>
+            )}
+            <Td textAlign='center' w={10}>
+              <Badge>{member.role}</Badge>
+            </Td>
+          </Tr>
+        ))}
+      </Tbody>
+    </Table>
+  </TableContainer>
+)
+
+// Component for active team members
+export const TeamMembersList = () => {
+  const { data: members, isLoading, isError, error } = useTeamMembers()
+
+  return (
+    <QueryDataLayout isEmpty={!members} isLoading={isLoading} isError={isError} error={error}>
+      <Flex direction='column'>
+        <Text fontWeight='bold' display='flex' gap={3}>
+          <Trans i18nKey='team.team_members'>Team members</Trans>
+          <Badge ml='1' colorScheme='green'>
+            {members?.length}
+          </Badge>
+        </Text>
+        <TeamMembersTable members={members || []} />
+      </Flex>
+    </QueryDataLayout>
+  )
+}
+
+// Component for pending team members
+export const PendingTeamMembersList = () => {
```

Comment:
Where is this component used?

Reply:
Oh I see there was an error and I was importing TeamMembersList rather than this other TeamMembers component.. so you weren't seeing the pending members at all 😅